mne/decoding/transformer.py

```diff
@@ -30,23 +31,28 @@ class Scaler(TransformerMixin):
 
     Attributes
     ----------
-    `scale_` : dict
+    info : instance of Info
+        The measurement info
+    ch_mean_ : dict
         The mean value for each channel type
-    `ch_std_` : array
+    std_ : dict
```

Comment:
I just changed the doc here, because the attribute is std_
We discussed about this class during the sprint, attributes names are messed up.